### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sjib**

### CODE REVIEW

**Review by ponceta — APPROVED** [review]
As soon as I have your green light @sjib I'll merge this so we have something at least.

**Comment by sjib** [comment]
@ponceta Great - one point:
 - TWW plugin version
 - TWW datamodel version
 shouldn't they be the same in tww? 2024.0, 20204.1 etc.
Do we still have to distinguish that?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+labels: Feature
+assignees: ''
+
+---
+
+## General information
+
+- Applicant:
+- Developer:
+- User group referent:
```
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: Who is that? Contact person in user group? Do we still need that? Or what would be helpful here?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- Developments referent:
```
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]: Same question here

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+### Risks and constraints
+
+
+### Mutualisation
+_Mutualisation and possible impacts with other projects_
+
+
+## Specification and offers ( Developer )
+Not shared on Github.
```
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:56]: Maybe just set 
- [ ] specification exist
- [ ] offers handed in 
- [ ] offer accepted
or we completely take this out here.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+### Specifications
+
+### Tenders
+
+Not shared on Github.
```
Comment (sjib) [inline .github/ISSUE_TEMPLATE/feature_request.md:62]: see offers

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (ponceta) [reply .github/ISSUE_TEMPLATE/feature_request.md:14]: Applicant -> provider
Developer -> developer
Client -> end user

**Comment by sjib** [comment]
@ponceta Lets start like this

**Pull request merged**